AngularJS/src/org/angular2/entities/metadata/stubs/Angular2MetadataDirectiveStubBase.java

```diff
@@ -115,4 +204,14 @@ private void loadAdditionalBindingMappings(@NotNull Map<String, String> mappings
       stubDecoratorFields(initializer, propertyName);
     }
   }
+
+  private static <T extends JsonValue> Function<JsonObject, T> toPropertyValue(
+        @NotNull final String property,
+        @NotNull final Class<T> clazz) {
+    return o -> getPropertyValue(o.findProperty(property), clazz);
+  }
+
+  private static <T extends JsonValue> Function<Object, T> toCast(@NotNull final Class<T> clazz) {
```

Comment:
I think this can be removed if using `select`

Reply:
@piotrtomiak yes, absolutely!